edit.js

```diff
@@ -1535,6 +1535,23 @@ document.getElementById('avatar-drop-zone').addEventListener('drop', async e =>
     await loginManager.setAvatar(dataHash);
   }
 });
+document.getElementById('packages-search').addEventListener('input', e => {
+  const searchTerm = e.target.value.toLowerCase();
+  const packages = [...document.querySelectorAll('#packages .package')];
+
+  if (!searchTerm) {
+    packages.forEach(p => (p.style.display = 'block'));
+    return;
```

Comment:
I am generally against this early-return style for logic for things that aren't "guard conditions".

IMO there is only one good use case for early returns, and that is for checking preconditions for whether we want to continue with the logic of the method.

Otherwise, I would stick to nested branches where it's clear what chain of logic got to the end result.